conda_build/post.py

```diff
@@ -180,9 +187,12 @@ def compile_missing_pyc(files, cwd, python_exe):
             call([python_exe, '-Wi', '-m', 'py_compile', f], cwd=cwd)
 
 
-def post_process(files, prefix, config, preserve_egg_dir=False):
+def post_process(files, prefix, config, preserve_egg_dir=False, noarch=False):
     rm_pyo(files, prefix)
-    compile_missing_pyc(files, cwd=prefix, python_exe=config.build_python)
+    if not noarch:
```

Comment:
I'd flip this around.  Logic is simpler when not negated.

```
if noarch:
    rm_pyc()
else:
    compile_missing_pyc
```

The argument against might be that it is a less common code path, and is thus a little slower.  I don't think this is a critical speed path.